src/lib.rs

```diff
@@ -223,8 +223,8 @@ impl Mnemonic {
 		}
 
 		Ok(Mnemonic {
-			lang: language,
-			words: words,
+			lang,
+			words,
```

Comment:
Also here please follow the style of the other methods and don't convolute field names with variables names.